roles/tftp-server/defaults/main.yml

```diff
@@ -1,14 +1,17 @@
 ---
 netboot_image: https://deb.debian.org/debian/dists/stretch/main/installer-amd64/current/images/netboot/netboot.tar.gz
-late_command_url: https://anonscm.debian.org/cgit/debconf-video/ansible.git/plain/setup_ansible.sh
+# late_command_url: https://anonscm.debian.org/cgit/debconf-video/ansible.git/plain/setup_ansible.sh
+late_command_url: http://{{ inventory_hostname }}/lc/late_command.sh
```

Comment:
That `lc` directory has always grated me. Having a directory with all the pxe related stuff would make sense. A directory just for one file is silly.